### ISSUE TITLE
Can I submit metrics using client token? (intead of api_key/app_key)

### ISSUE THREAD

**Issue by aguynamedben** [brettlangdon/node-dogapi#76]
Thanks for building this great library!

Hi, I'm building an Electron app and I'd like to collect metrics from the app. Is there any way I can use a [Client Token](https://docs.datadoghq.com/account_management/api-app-keys/) to submit metrics? I'm playing with the API but keep getting erros that api_key is required.

The log management system and RUM can take a client token, but I haven't found a way to get their metrics APIs to accept a client token.

**Comment by narwold** [comment]
I would very much like this too.

**Comment by aguynamedben** [comment]
Yeah!!! Datadog would become super engrained at our company if we could send off metrics from our app. Right now it's a glorified log collector. We use some of the log processing features to generate faux-metrics, but I wish I could send off real metrics from our app.

🙏

**Comment by brettlangdon** [comment]
Unfortunately you won't be able to use a client token for this.

This is just a client for the API which requires an API and possibly an APP
token.

I would suggest reaching support@datadoghq.com to explain your use case and
see what the best options are!

**Comment by narwold** [comment]
What I've found is that the browser SDK can send logs via a client token. Not sure about metrics. Our use case is that we want a limited key/token that we can use to send some data. We'd prefer metrics, but possibly we could send logs and parse metrics out of it. We don't want to distribute an all-powerful API token across our dev machines, but we're trying to collect app build performance metrics from devs machines to measure developer experience. It seems there's no way to restrict an API key, so I was going down the path of using a client token. However, I can't get the browser SDK to work outside the browser (Node in our case), so I'm stuck trying to reverse engineer the RUM API calls without any documentation on the RUM intake API being available that I can find. Any suggestions would be welcome.
